plugins/header_rewrite/conditions.cc

```diff
@@ -86,9 +86,8 @@ ConditionMethod::eval(const Resources &res)
   std::string s;
 
   append_value(s, res);
-  bool rval = static_cast<const Matchers<std::string> *>(_matcher)->test(s);
-  TSDebug(PLUGIN_NAME, "Evaluating METHOD(): %s - rval: %d", s.c_str(), rval);
-  return rval;
+  TSDebug(PLUGIN_NAME, "Evaluating METHOD()");
```

Comment:
Why did you drop the value from this debug message? The one below in ``ConditionHeader`` still has it ...